.vscode/tasks.json

```diff
@@ -17,8 +17,11 @@
         "panel": "shared"
       },
       "command": "make",
-      "args": ["clean", "example-crashpad-mac"],
+      "args": [
+        "clean",
+        "example-crashpad-mac"
```

Comment:
reminds me, we can probably just remove this file? At least this make target is invalid by now.

Reply:
Sure ! We might also delete launch.json and c_cpp_properties.json the same way.